src/GeneralStateTestsFiller/stEWASMTests/callSenderBalanceExceeds128BitsFiller.yml

```diff
@@ -0,0 +1,179 @@
+              (get_local $callDataSize)
+            )
+
+            ;; store call data
+            (call $storageStore
+              (get_local $ptrStorageKey11)
+              (get_local $ptrCallData)
+            )
+          )
+        )
+    deadbeef00000000000000000000000000000000:
+      balance: 100000000000
+      nonce: ''
+      storage: {}
+      code: |
+        (module
+          (import "ethereum" "call" (func $call (param i64 i32 i32 i32 i32) (result i32)))
+          (import "ethereum" "storageStore" (func $storageStore (param i32 i32)))
+
+          (memory 1)
+          (data (i32.const 0) "\de\ad\be\ef\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\01")
+          ;; init mem
+          (data (i32.const 32) "\01\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00") ;; 1
+          (export "memory" (memory 0))
+          (export "main" (func $main))
+
+          (func $main
+            ;; locals
+            (local $memStart i32)
+            (local $ptrStorageKey1 i32)
+            (local $ptrValue i32)
+            (local $ptrData i32)
+            (local $ptrResult i32)
+
+            ;; init
+            (set_local $memStart (i32.const 32))
+
+            ;; init pointers
+            (set_local $ptrStorageKey1 (get_local $memStart))
+            (set_local $ptrValue (i32.add (get_local $memStart) (i32.const 32)))
+            (set_local $ptrData (i32.add (get_local $memStart) (i32.const 64)))
+            (set_local $ptrResult (i32.add (get_local $memStart) (i32.const 96)))
+
+            ;; init mem
+            (i32.store (get_local $ptrValue) (i32.const 0xbeef))
+            (i32.store (get_local $ptrData) (i32.const 0))
+
+            ;; call CALL with data and save result
+            (i32.store
+              (get_local $ptrResult)
+              (call $call
+                ;; gas limit
+                (i64.const 200000)
+                ;; address offset (stored in data section above)
+                (i32.const 0)
+                ;; value offset
+                (get_local $ptrValue)
+                ;; data offset
+                (get_local $ptrData)
+                ;; data length
+                (i32.const 0)
+              )
+            )
+
+            ;; store the result
+            (call $storageStore
+              (get_local $ptrStorageKey1)
+              (get_local $ptrResult)
+            )
+          )
+        )
+  expect:
+    - indexes:
+        data: !!int -1
+        gas: !!int -1
+        value: !!int -1
+      network:
+        - ALL
+      result:
+        a94f5374fce5edbc8e2a8697c15331677e6ebf0b:
+          # Balance should be updated to deduct gas cost
+          balance: 340282366920938463463374607431768168195
+        deadbeef00000000000000000000000000000000:
+          # Balance should be updated to deduct value in CALL
+          # balance = 100000000000 - 0xbeef (48879) = 99999951121
+          balance: 99999951121
```

Comment:
The balance of this contract, since this is the one doing a value transfer to `deadbeef00..01`, should exceed >128 bits.